vimv

```diff
@@ -4,7 +4,7 @@
 # USAGE: vimv [file1 file2]
 # https://github.com/thameera/vimv
 
-declare -r FILENAMES_FILE="$(mktemp --tmpdir vimv.XXX)"
+declare -r FILENAMES_FILE="$(mktemp)"
```

Comment:
Please try it with this. (your version creates the temp file in the current directory and show up in the files list)

```suggestion
declare -r FILENAMES_FILE=$(mktemp "${TMPDIR:-/tmp}/vimv.XXX")
```

Reply:
When I tried it on Linux it created a file in /tmp, or in $TMPDIR if it's not empty. What OS are you on?

Reply:
If you run it with a template (vimv.XXX) then the file created in the current directory. (it's easier to find the file if something goes wrong if it has vimv in the name)